mne/epochs.py

```diff
@@ -1233,6 +1242,15 @@ def __getitem__(self, key):
             key = [key]
 
         if isinstance(key, (list, tuple)) and isinstance(key[0], string_types):
+            if any(["/" in k_i for k_i in epochs.event_id.keys()]):
```

Comment:
we should btw avoid any([list comp]) and all([list comp]) these work as generator expressions and will expand. Why I earlier believed this is a proble was due to pylab mode where any and all get overwritten by numpy functions. But that's for another issue maybe ...